src/lib/Libifl/pbs_loadconf.c

```diff
@@ -66,6 +66,8 @@
 
 char *pbs_conf_env = "PBS_CONF_FILE";
 
+#define LARGE_BUF_LEN 4096
```

Comment:
Is there a common header where you can move? You used it in cmd.h also. You could remove double definitions.